webview_app/lib/webview_page.dart

```diff
@@ -73,14 +73,14 @@ class _WebViewPageState extends State<WebViewPage> {
     super.dispose();
   }
 
-  void createBaseUrl() {
-    if (widget.upiApps.isNotEmpty) {
-      baseUrl =
-          'https://${widget.env}checkout.boxpay.tech/?token=${widget.token}&hmh=1&${widget.upiApps}';
-    } else {
-      baseUrl =
-          'https://${widget.env}checkout.boxpay.tech/?token=${widget.token}&hmh=1';
-    }
+  void createBaseUrl() {   
+      if (widget.upiApps.isNotEmpty) {
+        baseUrl =
+            'https://${widget.env}checkout.boxpay.tech/?token=${widget.token}&hmh=1&${widget.upiApps}';
```

Comment:
why is domain not configured for this like the api urls? For prod. checkout url is https://checkout.boxpay.in